storage/src/s3/integrity_checker.cpp

```diff
@@ -23,6 +23,7 @@
 #include "bcstatetransfer/BCStateTran.hpp"
 #include "KeyfileIOUtils.hpp"
 #include "direct_kv_storage_factory.h"
+#include "Digest.hpp"
```

Comment:
If header file doesn't have it included why cpp file required it?

Reply:
`Digest.hpp` inclusion removed.